scipy-1.0/minimize.tex

```diff
@@ -0,0 +1,118 @@
+\subsubsection*{Solving nonlinear optimization problems}
+
+\begin{table}[]
+  \centering
+  \begin{tabular}{cccccccccc}
+      & \rotatebox{60}{\texttt{Nelder-Mead}} & \rotatebox{60}{\texttt{Powell}} & \rotatebox{60}{\texttt{CG}} & \rotatebox{60}{\texttt{BFGS}}& \rotatebox{60}{\texttt{Newton-CG}} & \rotatebox{60}{\texttt{dogleg}} & \rotatebox{60}{\texttt{trust-ncg}} & \rotatebox{60}{\texttt{trust-exact}} & \rotatebox{60}{\texttt{trust-krylov}} \\
+    \hline
+    Version added & $<$0.7 & $<$0.7 & $<$0.7 & $<$0.7 & $<$0.7 & 0.13 & 0.13 & 0.19 & 1.0 \\
+    \hline
+    It is a wrapper? &  &  & & &  &  & & & \cmark \\
+    \hline
+    \nth{1} derivatives &  & & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark & \cmark \\
+    \hline
+    \nth{2} derivatives &  &  &  & Approx. & \cmark & \cmark & \cmark & \cmark & \cmark \\
+    \hline
+    \makecell{Iterative Hessian \\
+    factorization} & & & & & \cmark &  & \cmark &  & \cmark \\
+    \hline
+    Local convergence& & & L & S & S & Q & S & Q & ?  \\
+    \hline
+    Global convergence & & &  & \cmark & \cmark & \cmark & \cmark & \cmark & ?  \\
+    \hline
+    \makecell{Line-search (LS) or\\ trust-region (TR)} & Neither  & LS & LS & LS & LS & TR & TR & TR & TR \\
+    \hline
+    References & \cite{nelder_simplex_1965, wright_direct_1996} & \cite{powell_efficient_1964} &
+      \cite{polak_note_1969, nocedal_numerical_2006} & \cite{nocedal_numerical_2006} & \cite{nocedal_numerical_2006}  &
+      \cite{powell_new_1970, nocedal_numerical_2006} &  \cite{steihaug_conjugate_1983, nocedal_numerical_2006} &
+      \cite{conn_trust_2000, more_computing_1983} & \cite{gould_solving_1999, lenders_trlib:_2016} \\
+    \hline
+  \end{tabular}
+  \caption{Optimization methods from \texttt{minimize} for unconstrained minimization.  The field \textit{It is a wrapper?}
+    informs if the implementation available in SciPy is a wrapper to a function written in other programming language
+    (e.g. C or FORTRAN). The fields \textit{\nth{1}} and \textit{\nth{2} derivatives}
+    informs if the first or second order derivatives are required. When \textit{\nth{2} derivatives} is flagged
+    with the word \textit{Approx.} it means that, while the algorithm does not requires the second-order derivatives from
+    the user, it computes an approximation internally and uses it to accelerate the method convergence.
+    \textit{Local convergence} gives
+    a lower bound to the rate of convergence of the iterations sequence once the iterate is close enough to the
+    objective function solution. The convergence rates mentioned at the table
```

Comment:
"objective function solution" -> "solution", right? Literally, a "function" doesn't have a "solution" unless by solution you mean "root". You mean the solution to the optimization/nonlinear programming problem, and I think that "solution" is clear enough.
"mentioned at the table" doesn't sound natural (again, to a A-EN speaker). It would be "mentioned in the table", but "mentioned" is probably not the right word for technical writing. I'd suggest just dropping this phrase entirely: "The convergence rates are, from slowest to fastest:"

Reply:
Ok